src/test/java/csvDataDriven/CSVExercise.java

```diff
@@ -0,0 +1,66 @@
+package csvDataDriven;
+
+import au.com.bytecode.opencsv.CSVReader;
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CSVExercise {
+    WebDriver driver;
+
+    public static String[][] readCsvFile() throws IOException {
+        CSVReader csvReader = new CSVReader(new FileReader("src/test/java/csvDataDriven/Prices.csv"), ',', '"', 1);
+        String[] line;
+        List<String[]> userList = new ArrayList<>();
+        while ((line = csvReader.readNext()) != null) {
+            userList.add(line);
+        }
+        int rows = userList.size();
+        int cols = userList.get(0).length;
+        String[][] dataFromCSV = new String[rows][cols];
+        for (int i = 0; i < rows; i++) {
+            String[] eachRow = userList.get(i);
+            for (int j = 0; j < cols; j++) {
+                dataFromCSV[i][j] = eachRow[j];
+            }
+        }
+        return dataFromCSV;
+    }
+
+    @BeforeEach
+    public void setup() {
+        System.setProperty("webdriver.chrome.driver", "resources/windows/chromedriver.exe");
```

Comment:
@andreea-qa you can use [WebDriverManager](https://github.com/nadvolod/selenium-java/blob/0fe28d92ac3d0a037d267451acee584b01f16b54/src/test/java/HelloSelenium2Test.java#L12) with this [package](https://github.com/nadvolod/selenium-java/blob/0fe28d92ac3d0a037d267451acee584b01f16b54/pom.xml#L40)